docs/consensus/pot.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Proof-of-Time
+sidebar_position: 5
+description: Proof-of-Time consensus component
+keywords:
+    - Consensus
+    - randomness
+    - challenge
+---
+In addition to Proof-of-Space component described in previous section, Dilithium is secured by a Proof-of-Time (PoT) component. The chosen PoT algorithm is sequential AES, tuned for 1 proof per second. PoT is a measure against long-range attacks, and addresses unpredictability and dynamic availability issues. 
+The permissionless Proof-of-Work (PoW) used by Bitcoin remains the most robust consensus method in decentralized systems. However, mining requires massive energy expenditure to brute-force hash solutions, which has led to the introduction of various alternative consensus mechanisms. When transitioning to energy-efficient proof systems, like Proof-of-Stake and Proof-of-Space, however, several issues arise. 
+
+## Long-range attack
+
+Unlike in Proof-of-Work, the process of block production in Proof-of-Stake and Proof-of-Space-based
+blockchains is not physically constrained. This makes such protocols vulnerable to a _long-range attack_, when an attacker can produce, very quickly a chain all the way from genesis to the current time, and this chain can potentially be heavier than the current "canonical" chain.
```

Comment:
```suggestion
blockchains is not physically constrained. This makes such protocols vulnerable to a _long-range attack_, where an attacker can produce, very quickly, an alternative chain all the way to the current time, and this chain can potentially be heavier than the current "canonical" chain.
```
removed "from genesis", because it is not necessary in the attack.